test/eip20/eip20Factory.js

```diff
@@ -1,11 +1,87 @@
 
 contract('EIP20Factory', (accounts) => {
-  it('Verify a Human Standard Token once deployed using both verification functions.', async () => {
+  it('Verify an EIP20 token once deployed using both verification functions.', async () => {
+    const initialAmount = 100000;
+    const name = 'Simon Bucks';
+    const decimals = 2;
+    const symbol = 'SBX';
+    const args = [initialAmount, name, decimals, symbol, { from: accounts[0] }];
+
+    // new instance
+    const factory = await EIP20Factory.new();
+
+    // simulate: factory create a new EIP20 token
+    const newTokenAddr = await factory.createEIP20.call(...args);
+
+    // tx: factory create a new EIP20 token
+    await factory.createEIP20(...args);
+
+    // verify: new token's bytecode === EIP20's bytecode
+    const result = await factory.verifyEIP20.call(newTokenAddr, { from: accounts[0] });
+    assert.strictEqual(result, true, 'the bytecode at newTokenAddr '
+      + 'was not the same as the bytecode of an EIP20 token');
+
```

Comment:
for consistency, would add `// verify: new token's address is listed in the `isEIP20` mapping`